pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -979,9 +976,10 @@ func (c InstallerController) Sync(ctx context.Context, syncCtx factory.SyncConte
 
 	// Only manage installation pods when all required certs are present.
 	if err == nil {
-		requeue, syncErr := c.manageInstallationPods(ctx, operatorSpec, operatorStatus)
+		requeue, after, syncErr := c.manageInstallationPods(ctx, operatorSpec, operatorStatus)
```

Comment:
could we drop `requeue` param and requeue when `after.IsNotZero` ?

Reply:
it looks like `0` is a valid input to `AddAfter` it just means requeue immediately.